src/game/server/swarm/asw_harvester.cpp

```diff
@@ -42,6 +42,7 @@ ConVar asw_harvester_new( "asw_harvester_new", "1", FCVAR_CHEAT, "If set to 1, u
 ConVar asw_harvester_spawn_height( "asw_harvester_spawn_height", "16", FCVAR_CHEAT, "Height above harvester origin to spawn xenomites at" );
 ConVar asw_harvester_spawn_interval( "asw_harvester_spawn_interval", "1.0", FCVAR_CHEAT, "Time between spawning a harvesite and starting to spawn another" );
 ConVar rd_harvester_health( "rd_harvester_health", "200", FCVAR_CHEAT, "Health of the harvester" );
+ConVar rd_harvester_always_release_xenomites( "rd_harvester_always_release_xenomites", "0", FCVAR_NONE, "If set, harvesters will always release xenomites after death" );
```

Comment:
```suggestion
ConVar rd_harvester_always_release_xenomites( "rd_harvester_always_release_xenomites", "0", FCVAR_CHEAT, "If set, harvesters will always release xenomites after death" );
```

Challenge convars should be FCVAR_CHEAT.